app/views/project_content_items/_flags.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="modal-dialog" role="document">
+  <div class="modal-content">
+    <div class="modal-header">
+      <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+      <h4 class="modal-title">Flag '<%= content_item.title %>' for review</h4>
+    </div>
+    <div class="modal-body flag-for-review">
+      <%= form_for content_item, url: update_flags_project_content_item_path(project, content_item), method: :post, remote: true, html: { data: { module: 'radio-toggle' } } do |f| %>
+
+      <div class='radio'>
+        <label>
+          <%= f.radio_button :flag, "needs_help" %>
+          I need help tagging this
+        </label>
+      </div>
+
+      <div class='radio'>
+        <label>
+          <%= f.radio_button :flag, "missing_topic", data: { target: 'suggested-tags' } %>
+          There's no relevant topic for this
+        </label>
+      </div>
+
+      <div id='suggested-tags' class='form-group add-top-margin add-left-border'>
+        <%= f.label :suggested_tags, "Suggest a new topic" %>
+        <%= f.text_field :suggested_tags, class: 'form-control input-md-4' %>
+      </div>
+
+      <div class='modal-footer'>
+        <%= f.submit "Continue", class: 'btn btn-success add-right-margin' %>
+        <button type="button" class="btn btn-link" data-dismiss="modal">Cancel</button>
+      </div>
```

Comment:
If you move the modal-footer out of the modal-body, it looks better. I've pushed my change up here [1].

1: https://github.com/alphagov/content-tagger/commit/8766ff791d87dc13b573d1464bd75cf5cb4ff755